SUPPORT.md

```diff
@@ -0,0 +1,22 @@
+# Getting support
+
+This document explains how to get support if you're facing any issues with the Chat with your Data Solution Accelerator. Please read through the following guidelines.
+
+
+## Raise an issue
+If you've found a bug in Chat with your data, please [raise an issue](CONTRIBUTING.md#submit-issue) with us to get support.
+
+## Troubleshoot a problem
+You could also troubleshoot the issue yourself by accessing the Azure Workbook that gets deployed when you deploy Chat With Your Data to your Azure subscription. The Workbook contains logs and metrics from various components of the application (e.g. latency, request failure count, tokens processed by OpenAI service etc).
+
+To access the workbook, go to your resource group on the Azure portal and click on the Azure Workbook resource.
+![Azure portal - resource group](/media/resource-group.png)
```

Comment:
I'm wondering if we should just link to the MSFT docs instead? https://learn.microsoft.com/en-us/azure/azure-monitor/visualize/workbooks-overview#accessing-azure-workbooks